server/plugins/process-error-response.js

```diff
@@ -0,0 +1,14 @@
+function processErrorResponse (request) {
+  const response = request.response
+  const statusCode = response.output.statusCode
+  if (statusCode !== 404) {
+    request.log('error', {
+      statusCode: statusCode,
+      data: response.data,
+      message: response.message
+    })
+  }
+  return response

```

Comment:
We seem to no longer be rendering any errors here. It makes that we're not using views anymore as the service only outputs JSON but we do need to set appropriate headers. Do we do this elsewhere?

Reply:
This is all handled with the Hapi stack - when an error occurs the response returns the content type as `application/json; charset=utf-8`, the JSON contains a field with the error message, and the appropriate status code is returned in the header too.